libs/core/langchain_core/utils/env.py

```diff
@@ -31,10 +31,17 @@ def get_from_dict_or_env(
         return get_from_env(key, env_key, default=default)
 
 
-def get_from_env(key: str, env_key: str, default: Optional[str] = None) -> str:
+def get_from_env(
```

Comment:
lets not change for core - thats a bigger decicision

lets just do the string check outside this

Reply:
I revert the core changes and did the check outside as you suggest.